inscriptions/ISic002751.xml

```diff
@@ -191,21 +196,23 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI331799">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#FP">
                 <ab>
-                    <lb n="1"/>
-
-
-                    Στουδιῶ-<lb n="2"/>σε #⁵⁶ χρηστὲ<lb n="3"/>χαῖρε #⁵⁶ ἔζη-<lb n="4"/>σε #⁵⁶ ἔτη #⁵⁶ ξ ❦<lb n="5"/>
-                    λυδιαστής.
-                    <lb n="6"/>
+                    <lb n="1"/><persName type="attested"><name>Στουδιῶ
+                    <lb n="2" break="no"/>σε</name></persName> <g ref="#interpunct">·</g> <w>χρησ<hi rend="ligature">τὲ</hi></w>
+                    <lb n="3"/><w>χαῖρε</w> <g ref="#interpunct">·</g> <w>ἔζη
+                    <lb n="4" break="no"/>σε</w> <g ref="#interpunct">·</g> <w>ἔτη</w> <g ref="#interpunct">·</g> <num value="60"><hi rend="supraline">ξ</hi></num>  <g ref="#ivy-leaf">❦</g>
+                    <lb n="5"/><w>λυδιαστής</w>
```

Comment:
it's an interesting question whether we consider a craft/occupation as an 'office', but if we're going to tag carpenter(_tekton_) with roleName, then we should probably tag _ludiastes_ with roleName also (NB we can do so even when not part of the naming phrase). so I would put `<roleName>` around this final word.